src/utils/updaters/updateEventElementsStyle.ts

```diff
@@ -0,0 +1,12 @@
+import { CalendarInfo, ExtendedEvent } from "../../types.js";
+import { getFilteredExtEvents } from "../getters/getFilteredExtEvents.js";
+import { updateEventElementStyle } from "./updateEventElementStyle.js";
+
+export function updateEventElementsStyle(extEvent: ExtendedEvent, calendarInfo: CalendarInfo) {
```

Comment:
Naming is very similar to `updateEventElementStyle`, kind of hard to read the code, at first I thought this is some kind of a recursive function 😄 . Maybe name it something like `updateAllEventElementsStyles` or `bulkEventStylesUpdate`.